pennylane/measurements/measurements.py

```diff
@@ -615,34 +659,11 @@ def method_name(self, qscript: QuantumScript):
             '''
     """
 
-    method_name = ""
-    """Devices can override the logic of a measurement process by defining a method with the
-    name ``method_name`` of the corresponding class. The method should have the following signature:
-
-    .. code-block:: python
-
-        def method_name(self, qscript: QuantumScript):
-            '''Device's custom measurement implementation.
-
-            Args:
-                qscript: quantum script to transform
-            '''
-    """
-
-    method_name = ""
-    """Devices can override the logic of a measurement process by defining a method with the
-    name ``method_name`` of the corresponding class. The method should have the following signature:
-
-    .. code-block:: python
-
-        def method_name(self, qscript: QuantumScript):
-            '''Device's custom measurement implementation.
-
-            Args:
-                qscript: quantum script to transform
-            '''
-    """
-
     @abstractmethod
     def process(self, qscript, device):
-        """Process the given quantum script."""
+        """Process the given quantum script.
+
+        Args:
+            qscript (QuantumScript): quantum script to transform
+            device (Device): device used to transform the quantum script
+        """
```

Comment:
Can we have a usage example for `MeasurementTransform` as well?

Reply:
I am reluctant of adding an example here because this class is a temporary fix to be able to execute classical shadows as a CMP. With the new device API this might get deprecated.

Moreover, these types of measurements should use device's methods such as `reset` or `apply`, and I am not sure if we want to encourage users to use them.

Reply:
@josh146 what do you think?